src/main/java/uk/gov/hmcts/reform/prl/enums/ManageOrderFieldsEnum.java

```diff
@@ -29,7 +29,17 @@ public enum ManageOrderFieldsEnum {
     c21OrderOptions("c21OrderOptions"),
     selectChildArrangementsOrder("selectChildArrangementsOrder"),
     childArrangementsOrdersToIssue("childArrangementsOrdersToIssue"),
-    childOption("childOption");
+    childOption("childOption"),
```

Comment:
Are we supposed to remove childOption?

Reply:
Its not removed, we observed in one of the flow this field was throwing error but dint see the issue again. This field is not added/removed as part of this change so will take it separately